### ISSUE TITLE
installation errors

### ISSUE THREAD

**Issue by jacobmerson** [svalinn/parastell#37]
Based on the various errors we have worked through already, I think it would be helpful if you could provide a more detailed accounting of what versions of every package are you using as we have already found a number of incompatible python and library versions.

Also, we did not have any luck with the conda version of moab as it is missing some required headers for pymoab.

Here is an example error:
```
Traceback (most recent call last):
  File "/lore/mersoj/parastell/parastell/ExampleScript.py", line 1, in <module>
    import parastell
  File "/lore/mersoj/parastell/parastell/parastell.py", line 1, in <module>
    import magnet_coils
  File "/lore/mersoj/parastell/parastell/magnet_coils.py", line 1, in <module>
    import log
  File "/lore/mersoj/parastell/parastell/log.py", line 1, in <module>
    import logging
  File "/opt/scorec/spack/v0201_4/install/linux-rhel7-x86_64/gcc-7.4.0/python-3.10.10-sbgio2gf7lq3k3nvfslxe2uvwvkygwd3/lib/python3.10/logging/__init__.py", line 26, in <module>
    import sys, os, time, io, re, traceback, warnings, weakref, collections.abc
  File "/opt/scorec/spack/v0201_4/install/linux-rhel7-x86_64/gcc-7.4.0/python-3.10.10-sbgio2gf7lq3k3nvfslxe2uvwvkygwd3/lib/python3.10/re.py", line 124, in <module>
    import enum
  File "/opt/scorec/spack/v0201_4/install/linux-rhel7-x86_64/gcc-7.4.0/python-3.10.10-sbgio2gf7lq3k3nvfslxe2uvwvkygwd3/lib/python3.10/enum.py", line 2, in <module>
    from types import MappingProxyType, DynamicClassAttribute
  File "pymoab/types.pyx", line 1, in init pymoab.types
ImportError: numpy.core.multiarray failed to import (auto-generated because you didn't call 'numpy.import_array()' after cimporting numpy; use '<void>numpy._import_array' to disable if you are certain you don't need it).
```

**Comment by Edgar-21** [comment]
Here's a yml file of my currently working environment on Debian 5.10.209-2, it might work for you. 
[parastell.txt](https://github.com/svalinn/parastell/files/14312473/parastell.txt) (`conda env create -f parastell.yml`). Sorry to hear about the install issues. (looks like yml extensions aren't supported in github comments, not sure how much conda cares if the extension is right you may need to just change it back to yml after you download it.)

**Comment by Fuad-HH** [comment]
@jacobmerson produced the following `.yml` file  and it worked for us on RHEL-9.3.
```
name: parastellEnv
channels:
  - conda-forge
dependencies:
  - cadquery=2.3.1=pyhd8ed1ab_0
  - libnetcdf=4.9.1=nompi_h34a3ff0_101
  - moab=5.4.1=nompi_notempest_py311hb9f07ef_101
  - numpy=1.26.0=py311h64a7726_0
  - occt=7.7.0=h6cf717c_2
  - pybind11-abi=4=hd8ed1ab_3
  - scipy=1.11.3=py311h64a7726_1
  - matplotlib
  - pip:
      - cad-to-dagmc==0.5.0
      - cad-to-h5m==0.3.0
      - cftime==1.6.3
      - gmsh==4.11.1
      - netcdf4==1.6.5
      - networkx==3.2.1
      - trimesh==4.0.2
      - scikit-learn
```
Thank you.

**Comment by connoramoreno** [comment]
Closed by #69
